chapter2/tests/t20/terraform/dev/rds.tf

```diff
@@ -0,0 +1,36 @@
+module "wordpress_mysql_rds" {
+  source                = "terraform-aws-modules/rds/aws"
```

Comment:
all modules should be pinned to latest version at the time of code creation

Reply:
> all modules should be pinned to latest version at the time of code creation

@dmitry-mightydevops
Done